docs/reference/settings.md

```diff
@@ -185,6 +185,8 @@ By default, uv will resolve for all possible environments during a `uv lock` ope
 However, you can restrict the set of supported environments to improve performance and avoid
 unsatisfiable branches in the solution space.
 
+This follows [the grammar](https://packaging.python.org/en/latest/specifications/dependency-specifiers/#dependency-specifiers) originally specified in PEP 508.
```

Comment:
Separately, this seems confusing as we're not using the dependency specifier grammar, but the markers? https://packaging.python.org/en/latest/specifications/dependency-specifiers/#environment-markers

Reply:
You're right, the link I've put in is insufficiently specific. I'll update along with moving this to the relevant bit of code as per your other comment